test/src/derive.rs

```diff
@@ -328,15 +328,9 @@ impl Pcr0 {
                 input.rom_verify_config as u8,
             ],
         );
-        extend(
-            &mut value,
-            swap_word_bytes(&input.vendor_pub_key_hash).as_bytes(),
-        );
-        extend(
-            &mut value,
-            swap_word_bytes(&input.owner_pub_key_hash).as_bytes(),
-        );
-        extend(&mut value, swap_word_bytes(&input.fmc_digest).as_bytes());
+        extend(&mut value, input.vendor_pub_key_hash.as_bytes());
+        extend(&mut value, input.owner_pub_key_hash.as_bytes());
```

Comment:
We don't want to do this: input.owner_pub_key_hash is the words (in the weird hardware format); by calling as_bytes() we're actually reversing it from the regular sha384 serialization.

This could me made clearer in this test by making input.owner_pub_key_hash a [u8; 48] with the regular serialization.

Reply:
Basically, we never want to convert from [u32; 12] to [u8; 48] with zerocopy .as_bytes()....

Reply:
Ack, will close this PR